code/controllers/subsystem/job.dm

```diff
@@ -72,14 +72,7 @@ SUBSYSTEM_DEF(job)
 		if(!job)
 			JobDebug("AR job doesn't exists ! Player: [player], Rank:[rank]")
 			return FALSE
-		if(is_banned_from(player.ckey, rank))
-			JobDebug("AR isbanned failed, Player: [player], Job:[job.title]")
-			return FALSE
-		if(QDELETED(player))
-			JobDebug("AR player deleted during job ban check")
-			return FALSE
-		if(!job.player_old_enough(player.client))
-			JobDebug("AR player not old enough, Player: [player], Job:[job.title]")
+		if(!player.IsJobAvailable(rank,latejoin))
```

Comment:
This changes behavior, leave this for a different rework PR

Reply:
I disagree it changes any behaviour

Reply:
Well you contradicted  yourself :P It either doesn't change behavior, or it removes the manifest logs that the server is using and storing.